src/server.tsx

```diff
@@ -621,12 +621,24 @@ app.get(
 );
 
 app.post('/puzzles', async (req: express.Request, res: express.Response) => {
+    const { tracking } = req.body;
+    // Exclude AB test & campaign properties that relate to the admin console; we don't care about them for puzzles
+    const puzzlesTracking: Partial<BannerTestTracking> = {
+        componentType: 'ACQUISITIONS_PUZZLES_BANNER',
```

Comment:
I think this will be an issue, I'm pretty sure `Ophan` only ingests tracking if the `componentType` value matches a one of a set of allowed values, these are listed in their thrift definition here: https://github.com/guardian/ophan/blob/69a83e90e421cc610ee28fdcfb1bd8df451f1905/event-model/src/main/thrift/componentevent.thrift#L79

The process for updating these definitions is multi-stepped and a bit arduous from what I recall, what `componentType` do we use for regular Reader Revenue Banners? I think we should use the same here.

Reply:
OK I see we use `ACQUISITIONS_SUBSCRIPTIONS_BANNER`, I wonder whether we can reuse this componentEvent 🤔  @rupertbates can you an issue with this?

Reply:
There's also an `ACQUISITIONS_OTHER` which we seem to use quite often, so I actually think this may be the best option.

Reply:
👍 I've switched to `ACQUISITIONS_OTHER`